components/candle/challengeService.ts

```diff
@@ -371,12 +371,9 @@ export class ChallengeService extends ChallengeRegistry {
         const location = locations.children[child]
         assert.ok(location)
 
-        let contracts =
-            child === "LOCATION_AUSTRIA" ||
-            child === "LOCATION_SALTY_SEAGULL" ||
-            child === "LOCATION_CAGED_FALCON"
-                ? this.controller.missionsInLocations.sniper[child]
-                : this.controller.missionsInLocations[child]
+        let contracts = isSniperLocation(child)
```

Comment:
It's a shame the locations JSON doesn't have a field to designate it as a sniper level, this will prevent us from having to hardcode the location names.

Reply:
Yeah... but I guess it's the only way. Doubt they will add more sniper levels though.